common/interpolation/src/spline_interpolation.cpp

```diff
@@ -114,10 +114,12 @@ std::vector<double> splineByAkima(
     if (i == 0) {
       s_values.push_back(m_values.front());
       continue;
-    } else if (i == base_keys.size() - 1) {
+    }
+    if (i == base_keys.size() - 1) {
```

Comment:
Any reason to replace `else if` with `if` ?
I'm not sure yours is better than the current one.

Reply:
clang recommentation do not use else if after continue.